content/en/tracing/trace_retention_and_ingestion/usage_metrics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Usage Metrics
+kind: documentation
+description: "Learn how to monitor your Tracing without Limits usage."
+---
+
+## Usage Metrics Overview
+
+If, when you monitor your APM and Indexing usage, the numbers are not in line with your expectations, or you want to change your ingestion or indexing rates, see to the [Indexing](#indexing-controls) or [Ingestion](#ingestion-controls) documentation.
```

Comment:
```suggestion
If, when you monitor your APM and Indexing usage, the numbers are not in line with your expectations, or you want to change your ingestion or indexing rates, see the [Indexing](#indexing-controls) or [Ingestion](#ingestion-controls) documentation.
```

Reply:
These are not the right links. I think you mean to target a different page

Reply:
Ahhh hadnt updated this